.ci/template.windows.yml

```diff
@@ -0,0 +1,6 @@
+steps:
+  - bash: npm install eslint typescript@$(TS_VERSION) @typescript-eslint/eslint-plugin @typescript-eslint/parser eslint-config-prettier
+  - powershell: iwr https://deno.land/x/install/install.ps1 -out install.ps1; .\install.ps1 $(DENO_VERSION)
+  - bash: echo "##vso[task.prependpath]C:\Users\VssAdministrator\.deno\\bin"
+  - bash: npx eslint **/*.ts --max-warnings=0
```

Comment:
Can the eslint be moved to common?

Reply:
Nope, it's run with `npx` on Linux and windows, doesn't work on Mac :(